File: Python/lt_01_estseq/lt01_estseq_05.py
from math import pow, sqrt

axx=(int(input("Insira o Valor de ax²: ")))
bx=(int(input("Insira o Valor de bx: ")))
c=(int(input("Insira o Valor de c: ")))
delta=(pow(bx,2)-(4*(axx*c)))
print(delta)
if delta>0:
    c=((-bx+sqrt(delta))/(2*axx))
    delta=((-bx-sqrt(delta))/(2*axx))
    print("O valor de x' será ",c,".\nO Valor de x'' será ",delta,".")
elif delta==0:
    c=-bx/(2*axx)
    print("O valor de x será ",c,".")
else:
    delta*= -1
    axx*=2
    bx*=-1
    print("O valor de x' será ",bx,"/",axx,"+(i√",delta,")/",axx,".\nO valor de x'' será ",bx,"/",axx,"-(i√",delta,")/",axx,".")

File: C/lt_01_estseq/lt01_estseq_05.c
#include <math.h>
#include <stdio.h>
int main() {

  double axx, bx, c, delta, raiz;
  printf("Insita o valor de axx: ");
  scanf("%lf", &axx);
  printf("Insira o valor de bx: ");
  scanf("%lf", &bx);
  printf("Insira o valor de c: ");
  scanf("%lf", &c);
  delta = (bx * bx) - (4 * axx * c);

  if (delta >= 0) {
    raiz = sqrt(delta);
    c = (bx + raiz) / (2 * axx);
    delta = (bx - raiz) / (2 * axx);
    printf("O valor de X' e %.2lf\n", c);
    printf("O valor de x'' e %.2lf\n", delta);
  } else {
    printf("Valor inválido.\n");
  }
  return 0;
}
